tests/test_dataset.py

```diff
@@ -34,7 +34,7 @@ def test_dataset_valid_structure_200(dataset_test_data):
 
     mock_metadata_store = MagicMock()
     mock_metadata_store.get_dataset = MagicMock(return_value=dataset_test_data)
-    app.dependency_overrides[StubMetadataStore] = lambda: mock_metadata_store
+    app.dependency_overrides[OxigraphMetadataStore] = lambda: mock_metadata_store
```

Comment:
I think this needs to be changed back to StubMetadataStore until all of the Oxigraph methods are done (also in main.py).